mig/shared/defaults.py

```diff
@@ -76,6 +77,33 @@
 any_protocol = keyword_any
 any_state = keyword_any
 
+from pwd import getpwuid
```

Comment:
While the dynamic lookup makes sense for `generateconfs` etc. I have a slightly bad feeling about increasing dynamic constructs and actual code in `defaults.py`; it's really intended to be simple 'static' constants there, although we stretch it a bit in some cases. Slightly related to the keep-it-simple approach we talked about regarding unit tests.
Could we pull out this dynamic section and make it a helper e.g. in `mig.shared.base` or a new module instead? It could perhaps just take the 'static' confs dict and replace those vars accordingly.

I'm sure we could toggle the `enable_events` default to `False` to simplify, btw, but let's just keep that out of the mix for now.

Reply:
This was a great nudge - it prompted the realisation that, given how the rest of the changes make use of `keyword_auto`, we can probably lean on that here and kill two birds: remove the dynamic values creeping into defaults _and_ ensure we consistently use the same mechanism for value population.

Reply:
Made mig_user/mig_group use `keyword_auto`:
https://github.com/ucphhpc/migrid-sync/pull/72/files#diff-13b2a8d0b68628063fa8692dfa94986a499e335369a21e9234a9ebd1923ddc1dR82
https://github.com/ucphhpc/migrid-sync/pull/72/files#diff-aef9158f13f295cb08c3fe607c954b19860974733bc9f73b989ee583abec91e8R432